tools/run-test-perf.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+trap "exit" INT
+
+function pr_err() {
+  echo -e "\e[91mError: $@\e[39m"
+}
+
+function exit_err() {
+  pr_err $@
+  exit
+}
+
+USAGE="Usage:\n    sudo bash run.sh OLD_ENGINE NEW_ENGINE REPEATS"
+
+if [ "$(whoami)" != "root" ]
+then
+  pr_err "start script with sudo..."
+  echo -e $USAGE
+  exit
+fi
+
+if [ "$#" -ne 5 ]
+then
+  pr_err "Argument number mismatch..."
+  echo -e $USAGE
+  exit
+fi
+
+CPU=3
+ENGINE_OLD="$1"
+ENGINE_NEW="$2"
+REPEATS="$3"
+SINGLE_TEST_TIMEOUT="$4"
+TIMEOUT=$(echo "${REPEATS}" "${SINGLE_TEST_TIMEOUT}" | awk '{print $1 * $2;}')
+BENCH_FOLDER="$5"
+
+if [ ! -x $ENGINE_OLD ]
+then
+  exit_err "\"$ENGINE_OLD\" is not an executable file..."
+fi
+
+if [ ! -x $ENGINE_NEW ]
+then
+  exit_err "\"$ENGINE_NEW\" is not an executable file..."
+fi
+
+if [ $REPEATS -lt 1 ]
+then
+  exit_err "REPEATS must be greater than 0"
+fi
+
+if [ $SINGLE_TEST_TIMEOUT -lt 1 ]
+then
+  exit_err "SINGLE_TEST_TIMEOUT must be greater than 0"
+fi
+
+RAND_VA_SPACE="/proc/sys/kernel/randomize_va_space"
+echo 0 > $RAND_VA_SPACE || exit_err "setting $RAND_VA_SPACE"
+
+renice -n -19 $$ || exit_err "renice failed"
+#cpufreq-set -g performance || exit_err "couldn't set performance governor"
+#cpufreq-set -c ${CPU} -f 500MHz  || exit_err "couldn't set frequency"
```

Comment:
Could you, please, add script parameter that would specify whether to fix frequency to a passed value?